core/src/main/java/nl/nn/adapterframework/pipes/ForEachChildElementPipe.java

```diff
@@ -511,7 +511,7 @@ public void setElementXPathExpression(String string) {
 	}
 
 	/**
-	 * If set to <code>2</code> a XSLT processor 2.0 (net.sf.saxon) will be used, supporting XPath 2.0, otherwise a XSLT processor 1.0 (org.apache.xalan), supporting XPath 1.0. N.B. Be aware that setting this other than 1 might cause the input file being read as a whole in to memory, as XSLT Streaming is currently only supported by the XSLT Processor that is used for xsltVersion=1
+	 * If set to <code>2</code> a XSLT processor 2.0 (net.sf.saxon) will be used, supporting XPath 2.0, otherwise a XSLT processor 1.0 (org.apache.xalan), supporting XPath 1.0. N.B. Be aware that setting this other than 1 might cause the input file being read as a whole in to memory, as XSLT Streaming is currently only supported by the XSLT Processor that is used for xsltVersion=1. This pipe supports up to and including XSLT version 3.0
```

Comment:
Actually this comment should be updated to say that the Saxon processor will be used when setting this parameter to 2 or 3.